src/sections/Community/Handbook/programs-data.js

```diff
@@ -1,11 +1,17 @@
 import SCA from "../../../collections/programs/sca-contributhon/sca.svg";
+import SCALightlogo from "../../../collections/programs/sca-contributhon/sca-dark.svg";
 import MLH from "../../../collections/programs/mlh-2020/mlh.svg";
+import MLHLightlogo from "../../../collections/programs/mlh-2020/mlh-dark.svg";
 import LFX from "../../../collections/programs/lfx-2022/Linux_Foundation_logo.png";
+import LFXLightlogo from "../../../collections/programs/lfx-2022/lfx-white.svg";
 import Hacktoberfest from "../../../assets/images/programs/hacktoberfest.png";
 import GSOC from "../../../collections/programs/gsoc-2020/gsoc.svg";
 import GSOD from "../../../collections/programs/gsod/gsod.png";
 import Layer from "../../../assets/images/layer5/layer5-only/svg/layer5-white-bg.svg";
+import LayerLightlogo from "../../../assets/images/layer5/layer5-only/svg/layer5-dark-bg.svg";
 import Bridge from "../../../collections/programs/communitybridge/communitybridge-blue.png";
+import BridgeDark from "../../../collections/programs/communitybridge/communitybridge.png";
```

Comment:
prefer using .webp format files instead of png.

Reply:
Oh, I just used whatever files were available. I didn't add any. Only png files were available, so I used them.

Reply:
yeah no issue just replace them with webp